prep-javascript-dom-events/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Events</title>
+</head>
+
+<body>
+  <button class="hello-world-btn">Hello World!</button>
+  <script>
+    // Your code here
+    document.addEventListener('DOMContentLoaded', console.log('content loaded!'));
```

Comment:
This part is executing your console log before the document has finished loading, but we can defer that by wrapping it in a function.
```js
document.addEventListener('DOMContentLoaded', function () {
  console.log('content loaded');
});
```